src/components/views/dialogs/CreateGroupDialog.js

```diff
@@ -57,7 +57,7 @@ export default React.createClass({
         let error = null;
         if (!this.state.groupId) {
             error = _t("Community IDs cannot be empty.");
-        } else if (!/^[a-z0-9=_\-\.\/]*$/.test(this.state.groupId)) {
+        } else if (!/^[a-z0-9=_\-./]*$/.test(this.state.groupId)) {
```

Comment:
This doesn't seem right

Reply:
Current version
```
\. matches the character . literally (case sensitive)
\/ matches the character / literally (case sensitive)
```

Proposed version:
`./ matches a single character in the list ./ (case sensitive)`

Same thing